tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -77,6 +80,12 @@ func setupArguments() {
 		batchSize, _ = strconv.Atoi(batch)
 	}
 
+	randomShard := os.Getenv("RANDOM_SHARD_SIZE")
```

Comment:
I find the name `RANDOM_SHARD_SIZE` rather misleading. The shard size is not random, but a random shard ID between 0 and the provided value is assigned. Therefore I'd rather name it `SHARD_SIZE` or `RANDOM_SHARD_ID`, where the latter somehow indicates a boolean.

Reply:
I agree. It could just be `NUM_SHARDS` or `NUM_STREAM_SHARDS`?

Reply:
I don't disagree however this config was already supplied to a user and I don't want to change the name and break them in the future. Specifically because we will no longer need this when Loki can auto shard high volume streams automatically.

This is why I'm not really going to expose this in the docs.